assets/js/base/components/price-slider/index.js

```diff
@@ -300,20 +280,82 @@ const PriceSlider = ( {
 				) }
 			</div>
 			<div className="wc-block-price-filter__controls">
-				{ showInputFields ? (
-					<PriceInput
-						disabled={ isLoading || ! hasValidConstraints }
-						onChange={ priceInputOnChange }
-						onBlur={ priceInputOnBlur }
-						minPrice={ formattedMinPrice }
-						maxPrice={ formattedMaxPrice }
-					/>
-				) : (
-					<PriceLabel
-						minPrice={ formattedMinPrice }
-						maxPrice={ formattedMaxPrice }
-					/>
+				{ showInputFields && (
+					<Fragment>
+						<NumberFormat
```

Comment:
Using NumberFormat here for inputs and labels seemed more efficient than storing custom price strings and passing them around.

Reply:
Agreed.  If we implement the `FormattedPrice` price component idea I had, it could have custom events internally that automatically convert back and forth from the subunit values so your callbacks as implemented here deal with subunits (which keeps things cleanly separated from display and internal representation).  This will also have importance for potential interfaces where components using `FormattedPrice` inputs could be sending back data to the server over the wire (where values need to be subunits).